guw/main.py

```diff
@@ -80,21 +80,25 @@ def _sync_at(self, tmpdir, backup, local):
                 to_push.append((feature["name"], feature["remote"]))
                 prev_active_feature = feature
                 has_pending = True
+            else:
+                logger.critical(f"Feature {feature['name']} has unknown status: '{feature['status']}'")
+                return
             prev_feature = feature
         # Make target branch be the last feature
         last_feature = self.config["features"][-1]
         if last_feature:
             if last_feature["status"] != "integrated":
-                target_branch = f"{self.config['target']['remote']}/{self.config['target']['branch']}"
+                target_branch_name = self.config['target']['branch']
```

Comment:
The remote is no longer included in the checkout. I prefer to keep it explicit otherwise, it can collision with other branches

Reply:
@turran Are you sure?
Please, rereview more attentively and reread my commit message!

1. `target_branch` was never used. Deleted.
2. Independently of the previous, all complex expressions `self.config['target']['branch']` (or with double quotes) are substituted with variable that hold this expression.

Reply:
I see. Good to go then. I still want to keep an eye on the explicit checkout, not because of your commit but because of the current logic.